Package.swift

```diff
@@ -0,0 +1,32 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+import PackageDescription
+
+let package = Package(
+    name: "JitsiMeetSDK",
+    platforms: [.iOS(.v14)],
+    products: [
+        .library(
+            name: "JitsiMeetSDK",
+            targets: ["JitsiMeetSDKWrapper"]),
```

Comment:
What this wrapper all about?

Reply:
This wrapper is to make sure the dependency "WebRTC" and "GiphyUISDK" are well integrated.
I looked at different other `Package.swift` and it's usually how it's done.
You can have a look here : 
- https://github.com/sendbird/sendbird-calls-ios/blob/master/Package.swift
- https://github.com/Giphy/giphy-ios-sdk/blob/main/Package.swift

Reply:
TIL!